pom.xml

```diff
@@ -59,6 +59,7 @@
     <findbugs.maxRank>9</findbugs.maxRank>
     <hadoop.version>2.6.3</hadoop.version>
     <logback.version>1.1.3</logback.version>
+    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
It sounds like it may only redirect stdout to a file.  If the logging goes to stderr, it would not be redirected.  Personally I would rather fix the test than make this change.  When debugging issues I sometimes add temporary println to impl and test code and its nice to easily see it when running the test.

http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#redirectTestOutputToFile